src/main/java/com/frcteam3636/frc2024/subsystems/shooter/PivotIO.kt

```diff
@@ -249,7 +257,7 @@ class PivotIOKraken : PivotIO {
         const val PROFILE_JERK = 80.0
 
         val HARDSTOP_OFFSET: Rotation2d = Rotation2d.fromDegrees(-27.0)
-        val ABSOLUTE_ENCODER_OFFSET: Rotation2d = Rotation2d.fromDegrees(138.8) + HARDSTOP_OFFSET
+        var ABSOLUTE_ENCODER_OFFSET: Rotation2d = Rotation2d.fromDegrees(138.8) + HARDSTOP_OFFSET
```

Comment:
IMO this should stay a constant. It's in the Constants object so it makes sense for it to be a constant. Instead, we could have a separate class field that is initialized to that value as logged as an output.